WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -184,6 +185,17 @@ private void removeFollowedBlogsPreferenceForIfDisabled() {
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
+        boolean isLoggedIn = mAccountStore.hasAccessToken();
+        if (!isLoggedIn) {
+            // Not logged in users can start Notification Settings from App info > Notifications menu.
+            // If there isn't a logged in user, just show the entry screen.
+            Intent intent = new Intent(getContext(), WPLaunchActivity.class);
+            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
+            startActivity(intent);
+            getActivity().finish();
+            return;
```

Comment:
What does this **return** do here!

Reply:
I don't want to continue after closing the screen with `finish()`. Next lines (like  `loadNotificationsAndUpdateUI()`) works, if I don't add `return`.